documentation/scaling_puppet_server.markdown

```diff
@@ -95,28 +95,31 @@ If you [use `SRV` records for agents](#using-dns-srv-records), you can use the `
 
--   [Disable Puppet Server's certificate authority services](./configuration.markdown#service-bootstrapping).
+1.   [Disable Puppet Server's certificate authority services](./configuration.markdown#service-bootstrapping).
 
-    -   If you're using the [individual agent configuration method of CA centralization](#directing-individual-agents-to-a-central-ca), set `ca_server` in `puppet.conf` to the hostname of your CA server in the `[main]` config block.
-    -   If an `ssldir` is configured, make sure it's configured in the `[main]` block only.
+     If you're using the [individual agent configuration method of CA centralization](#directing-individual-agents-to-a-central-ca), set `ca_server` in `puppet.conf` to the hostname of your CA server in the `[main]` config block.
+    
+    If an `ssldir` is configured, make sure it's configured in the `[main]` block only.
 
--   If you're using the [DNS round robin method](#using-round-robin-dns) of agent load balancing, or a [load balancer](#using-a-load-balancer) in TCP proxying mode, provide compile masters with certificates using DNS Subject Alternative Names.
+2.   If you're using the [DNS round robin method](#using-round-robin-dns) of agent load balancing, or a [load balancer](#using-a-load-balancer) in TCP proxying mode, provide compile masters with certificates using DNS Subject Alternative Names.
 
-    -   Configure `dns_alt_names` in the `[main]` block of `puppet.conf` to cover every DNS name that might be used by an agent to access this master.
+     Configure `dns_alt_names` in the `[main]` block of `puppet.conf` to cover every DNS name that might be used by an agent to access this master.
 
         ```
         dns_alt_names = puppet,puppet.example.com,puppet.site-a.example.com
         ```
 
-    -   If the agent or master has been run and already created a certificate, remove it by running `sudo rm -r $(puppet master --configprint ssldir)`. If an agent has requested a certificate from the master, delete it there to re-issue a new one with the alt names: `puppetserver ca clean master-2.example.com`.
+     If the agent or master has been run and already created a certificate, remove it by running `sudo rm -r $(puppet master --configprint ssldir)`. If an agent has requested a certificate from the master, delete it there to re-issue a new one with the alt names: `puppetserver ca clean master-2.example.com`.
 
--   Request a new certificate by running `puppet agent --test --waitforcert 10`.
+3.   Request a new certificate by running `puppet agent --test --waitforcert 10`.
 
--   Log into the CA server and run `puppetserver ca sign master-2.example.com`.
+4.   Log into the CA server and run `puppetserver ca sign master-2.example.com`.
 
-    Add `--allow-dns-alt-names` to the command if `dns_alt_names` were in the certificate request.
+    If `dns_alt_names` were in the certificate request, add `--allow-dns-alt-names` to the command.
```

Comment:
I don't think this flag exists for `puppetserver ca`, https://tickets.puppetlabs.com/browse/SERVER-2318. I would just leave this recommendation off.

Reply:
done!